oam/workloads/application.meshery.layer5.io.schema.json

```diff
@@ -17,16 +19,20 @@
         },
         "labels": {
           "type": "object",
-          "additionalProperties": {
-            "format": "string",
-            "type": "string"
-          }    
+          "properties": {
+            "additionalProperties" : {
```

Comment:
can you attach a screenshot of how this changes the way RJSF renders additionalProperties ?

Reply:
![Screenshot from 2022-02-07 20-10-40](https://user-images.githubusercontent.com/65453826/152809376-3d4a6d2a-5c8f-44c9-914c-277f9f48be87.png)

Reply:
I'm not sure if this is what we want for the additionalProperties